### PR TITLE
Getting Started section

### PR DESCRIPTION
**Pull request by domsinclair**
Added a getting started section to the documentation.  Hopefully the markdown syntax is correct.  Possibly the images might need to be done  with a dark theme for the sake of contrast.

### CODE REVIEW

File: docs/guide/Getting-Started.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+Select XPO for the ORM and then click finish.
+
+
+
+![New Project 3 ](/images/guide/overview/newProject3.png)
+
+
+
+The wizard will then create the sution which, when finished,  will contain three projects.
```
Comment (Paramethod) [inline docs/guide/Getting-Started.md:39]: sution -> solution
Extra space between coma and will

Comment (domsinclair) [reply docs/guide/Getting-Started.md:39]: In my defence m'lud I can only cite being arisen from from slumbers by an irate client at a little after 3.0 am.  Will correct in time for next pr.

File: docs/guide/Getting-Started.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
+---
+title: Getting Started
+---
+# Getting Started
+
+
+Xenial.Framework supports DevExpress expressApplicationFramework (XAF) solutions that target both WinForms and Blazor and is available as a Nuget package. At the time of writing this documentation the minimum supported version of the XAF framework is Version 20.2.6 with the latest version of the Xenial.Framework being 0.0.54.
```
Comment (biohazard999) [inline docs/guide/Getting-Started.md:7]: We can use vue global variables for this and get the version numbers out of the build definiton, as well the supported version table.

Wait for my commit here.

Comment (domsinclair) [reply docs/guide/Getting-Started.md:7]: Great , anything that can be automated is always a boon.

**Comment by biohazard999** [comment]
Good work! Thank you!

**Pull request merged**